diff --git a/conf_drm.sh b/conf_drm.sh
index e01ea68..ab15fc2 100755
--- a/conf_drm.sh
+++ b/conf_drm.sh
@@ -1,5 +1,7 @@
 #!/bin/bash
 
+prefix=${PREFIX:-/usr}
+
 if test x$1 = x32; then
     is64bit=false
     archdir=i386-linux-gnu
@@ -14,7 +16,7 @@ rm -r build$1
 
 cflags="-fno-omit-frame-pointer"
 
-meson build$1 --prefix /usr --libdir /usr/lib/$archdir --buildtype debugoptimized \
-	-Dc_args=$cflags -Dc_link_args=$cflags -Dpkg_config_path=/usr/lib/$archdir/pkgconfig \
-	-Detnaviv=true -Dexynos=true -Dfreedreno=true -Domap=true -Dtegra=true -Dvc4=true \
-	-Dcairo-tests=$is64bit
+meson build$1 --prefix $prefix --libdir $prefix/lib/$archdir --buildtype debugoptimized \
+	-Dc_args=$cflags -Dc_link_args=$cflags -Dpkg_config_path=$prefix/lib/$archdir/pkgconfig \
+	-Detnaviv=disabled -Dexynos=disabled -Dfreedreno=disabled -Domap=disabled -Dtegra=disabled -Dvc4=disabled \
+	-Dcairo-tests=disabled
